api/v1beta2/configuration_types.go

```diff
@@ -96,12 +98,25 @@ type Property struct {
 	Value string `json:"value,omitempty"`
 }
 
-// Backend stores the state in a Kubernetes secret with locking done using a Lease resource.
+// Backend describes the Terraform backend configuration
 type Backend struct {
 	// SecretSuffix used when creating secrets. Secrets will be named in the format: tfstate-{workspace}-{secretSuffix}
+	// Deprecated: use the `type` and `config` instead
 	SecretSuffix string `json:"secretSuffix,omitempty"`
 	// InClusterConfig Used to authenticate to the cluster from inside a pod. Only `true` is allowed
+	// Deprecated: use the `type` and `config` instead
 	InClusterConfig bool `json:"inClusterConfig,omitempty"`
+
+	// HCL allows users to use raw hcl code to specify their Terraform backend configuration
+	HCL string `json:"hcl,omitempty"`
```

Comment:
We already have `HCL` https://github.com/oam-dev/terraform-controller/pull/291/files#diff-56b818bb103c20f38071e987a12aebf0914429db0e363f0ec2cd58a817967b4eR30. It might get mixed.

Reply:
Should we change `HCL` to something else, like `raw`?

Reply:
Yes. Maybe you can use `inline`.